gpu-operator/platform-support.rst

```diff
@@ -72,15 +72,18 @@ The following NVIDIA data center GPUs are supported on x86 based platforms:
      | | NVIDIA H200,          | NVIDIA Hopper             |
      | | NVIDIA H200 NVL       |                           |
      +-------------------------+---------------------------+
-     | NVIDIA HGX H200         | NVIDIA Hopper and         |
+     | NVIDIA DGX H100         | NVIDIA Hopper and         |
      |                         | NVSwitch                  |
      +-------------------------+---------------------------+
-     | NVIDIA DGX H100         | NVIDIA Hopper and         |
+     | NVIDIA DGX H200         | NVIDIA Hopper and         |
```

Comment:
NVIDIA DGX H200 should have Hopper and switch, correct?

Reply:
@cdesiniotis just wanted to confirm that this is correct here

Reply:
Yes, believe this is accurate